### ISSUE TITLE
ImportError for mxnet: cannot import name libinfo

### ISSUE THREAD

**Issue by QuantumLiu** [apache/incubator-mxnet#4794]
For bugs or installation issues, please provide the following information.
The more information you provide, the more likely people will be able to help you.

## Environment info
Operating System:
`Windows10`
Compiler:
`cl,nvcc,vs2013`
Package used (Python/R/Scala/Julia):
`Python`
MXNet version:
**0.9.3**
Or if installed from source:
installed from source
MXNet commit hash (`git rev-parse HEAD`):

If you are using python package, please provide

Python version and distribution:
`Python 2.7.12 |Anaconda 4.1.1 (64-bit)`
If you are using R package, please provide

R `sessionInfo()`:

## Error Message:
Please paste the full error message, including stack trace.
```
>>> import mxnet
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda2\lib\site-packages\mxnet-0.9.3-py2.7.egg\mxnet\__init__.py", line 7, in <module>
    from .base import MXNetError
  File "C:\Anaconda2\lib\site-packages\mxnet-0.9.3-py2.7.egg\mxnet\base.py", line 43, in <module>
    _LIB = _load_lib()
  File "C:\Anaconda2\lib\site-packages\mxnet-0.9.3-py2.7.egg\mxnet\base.py", line 35, in _load_lib
    lib = ctypes.CDLL(lib_path[0], ctypes.RTLD_GLOBAL)
  File "C:\Anaconda2\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126]
>>> import mxnet
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Anaconda2\lib\site-packages\mxnet-0.9.3-py2.7.egg\mxnet\__init__.py", line 7, in <module>
    from .base import MXNetError
  File "C:\Anaconda2\lib\site-packages\mxnet-0.9.3-py2.7.egg\mxnet\base.py", line 10, in <module>
    from . import libinfo
ImportError: cannot import name libinfo
>>>
```
## Minimum reproducible example
if you are using your own code, please provide a short script that reproduces the error.

## Steps to reproduce
or if you are running standard examples, please provide the commands you have run that lead to the error.

1.python setup.py install 
2.import mxnet
3.

## What have you tried to solve it?

1.I tried to run a edited setup.py file #4753,but still get error
2.
3.

**Comment by DamonDeng** [comment]
duplicate with #4746 .

My comment in #4746 

One of the reason for this issue is that you run the python command in the exact directory you install MXNet.

Say that you clone the MXNet project into /home/user1/incubator-mxnet, you build the mxnet library then go to the python directory to run install command python setup.py install --user.

After that , please exit the python directory before you run python to test mxnet.

As there is a directory named mxnet in the incubator-mxnet/python directory, if you run python in directory incubator-mxnet/python and then import mxnet as mx, python will try to import the directory instead of the mxnet lib.

In conclusion, do not start python shell in the incubator-mxnet/python directory, or else you will get the cannot import name libinfo error message.

Please create a new issue to report it if you didn't run python shell in incubator-mxnet/python while you get the same error message cannot import name libinfo

**Comment by szha** [comment]
Shouldn't be a problem any more in the newer versions. Feel free to reopen if issue persists
